conductr_cli/sandbox_run_jvm.py

```diff
@@ -185,6 +205,20 @@ def log_run_attempt(args, run_result, feature_results, feature_provided):
         for message in run_result.license_validation_error.messages:
             log.warning(message)
 
+    elif run_result.sandbox_upgrade_requirements and run_result.sandbox_upgrade_requirements.is_upgrade_required:
+        def format_version(version_info):
+            return semver.format_version(version_info.major,
+                                         version_info.minor,
+                                         version_info.patch,
+                                         version_info.prerelease,
+                                         version_info.build)
+
+        upgrade_requirements = run_result.sandbox_upgrade_requirements
+        latest_version_str = format_version(upgrade_requirements.latest_version)
+        log.info('')
+        log.warning('Please upgrade to sandbox version {} by running'.format(latest_version_str))
```

Comment:
What about changing the naming to:

```
log.info('')
log.warning('A newer ConductR version is available. Please upgrade the sandbox to {} by running'.format(latest_version_str))
log.warning('  sandbox run {}'.format(latest_version_str))
```

This messaging sounds less harsh and only provides the information that a newer version is available instead of requesting that the user needs to upgrade.